docs/orchestrating-elastic-stack-applications/logstash.asciidoc

```diff
@@ -874,6 +874,141 @@ spec:
 <1> Value of password to protect the Logstash keystore
 <2> The syntax for referencing keys is identical to the syntax for environment variables
 
+[id="{p}-logstash-custom-files-plugins"]
+== Adding plugins and files
```

Comment:
Good content here.  There's not a lot of shared content here, and it seems like it would be easier to read and follow if you broke this topic into two topics:

* Adding a plugin
** Adding a plugin using an initContainer
** Adding a plugin using a custom image

* Adding more files
** Adding files using an initContainer
** Adding files using a custom image

Reply:
The repetition in headings in intentional to provide the user with specific guideposts, and for better SEO.